tests/unit/async/Task.ts

```diff
@@ -147,6 +147,19 @@ let suite = {
 
 			resolver();
 		}
+	},
+
+	'Task#resolve': {
```

Comment:
I was following the other patterns in the file, but I guess it is a static method, versus the `#` I guess indicating an instance method.